themes/wdfn_theme/layouts/index.html

```diff
@@ -1,4 +1,13 @@
 {{ define "main" }}
+  <h2>Welcome to the experimental space of Water Data for the Nation!</h2>
```

Comment:
My understanding (based on a week of work with Hugo) is that we shouldn't touch anything inside of the themes folder. This is because the themes are updated through package management, and the updates will overwrite our changes. I believe it is possible to get around this by duplicating the theme file (and directory structure in the root directory). In this case duplicate (then add the desired changes to) the index.html as layout/index.html in the root directory. When Hugo receives the 'run build' command it will see the duplicate file and give it precedence over the copy in the theme. 

I'll poke  at this first thing in the morning and see if I can get the changes you want to occur outside the theme.

Reply:
Technically, in this case I think it would be OK because the theme is just part of the repo, it doesn't come from anywhere (versus having a separate theme to pull from from a specific theme repo).  However, since we probably want to move to having a shared WDFN/USGS theme that is used across multiple Hugo pages, it's probably not a bad idea to start using the layouts approach.

Reply:
Gotcha. I was thinking we had packaged the WDFN theme so that it was updated in the same was as the WDFN-vis. So, yup, seems that it would be fair to make the changes here.